extensions/value_generators/models/generators.py

```diff
@@ -31,23 +30,22 @@ class Copier(value_generators_domain.BaseValueGenerator):
 
     default_value = ''
 
-    def generate_value(self, context_params, value, parse_with_jinja=False):
+    def generate_value(self, unused_context_params, value, **kwargs):
         """Returns a copy of the input value.
 
-        If parse_with_jinja is True, strings within the input value are treated
-        as templates and parsed against context_params. The output will be a
-        unicode string.
-
-        If parse_with_jinja is False, the input value is copied and returned
-        without changing its type.
+        Args:
+            unused_context_params: dict. Context params parsed with input
+                value which is treated as a template. Not used.
+            value: str. Value whose copy should be returned.
+            **kwargs: dict. Contains parse_with_jinja which is a part
```

Comment:
I would say, don't use **kwargs -- in general we try to avoid that sort of indirection. How about just unused_parse_with_jinja?

Reply:
unused_parse_with_jinja would throw an error saying unexpected keyword argument `parse_with_jinja` right?
can we use 'parse_with_jinja' itself and add a statement to disable the pylint unused-argument for that line? wdyt?